contrib/ansible/vagrant/provision.sh

```diff
@@ -0,0 +1,9 @@
+echo "hello, here is  a sample provisioning script that demonstrates everything works"
+ls /vagrant
+### Required for ansible??
+### Otherwise 'no yum module installed'...
+sudo yum install -y yum # needed for the ansible python stuff
+sudo yum install -y docker
+sudo yum install -y openssl # needed for making certs
```

Comment:
actually, no, i think you are right...   put an openssl install somewhere in the roles/kubernetes/tasks/gen_certs.yml

check out roles/master/tasks/generic_install.yml for an example of how to install packages (in a yum/dnf/apt agnostic way)